### PR TITLE
Add create-vmci-socket.yml

### PR DESCRIPTION
**Pull request by jtothej**
Add create-vmci-socket.yml

### CODE REVIEW

File: communication/socket/create-vmci-socket.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+rule:
+  meta:
+    name: create VMCI socket
+    namespace: communication/socket
```
Comment (williballenthin) [inline communication/socket/create-vmci-socket.yml:4]: do you think this is most related to "socket" functionality or VM interaction?

Comment (williballenthin) [reply communication/socket/create-vmci-socket.yml:4]: im not too familar with this tech, but i wonder if the namespace should be more related to vm interaction, maybe like `hsot-interaction/virtual-machine/vmci` or something

**Review by williballenthin — APPROVED** [review]

File: communication/socket/create-vmci-socket.yml (hunk 1 above)
Comment (jtothej) [reply communication/socket/create-vmci-socket.yml:4]: It's a good point and I think that VMCI sockets could fit in both categories/namespaces but ultimately they are just another socket type so `communication/socket` (or `communication/socket/vmci`) seems to be more adequate.

Comment (williballenthin) [reply communication/socket/create-vmci-socket.yml:4]: cool, thanks for confirming!

**Review by mr-tz — APPROVED** [review]
nice, thanks!

**Pull request merged**